src/js/preshuffle.s

```diff
@@ -908,37 +937,31 @@ UpdatePlayerMaxHPAndMPAfterLevelUp:
   clc
   adc PlayerMP
   sta PlayerMP
-  jmp UpdateDisplayAfterLevelUp
+  rts
 
-.org $91c7
+.reloc
 UpdateDisplayAfterLevelUp:
+  jsr UpdateHPDisplayInternal
+  lda #$19 ; 0b0001'1001 ie: 0, 3, 4
+  jsr UpdateStatusBarDisplays
+  lda #GAME_MODE_STATUS_MSG
+  sta GameMode
+  lda #$0d
```

Comment:
These two stores could use some comments?  Especially since I have no idea (off the top of my head) what $6c3 and $6e3 are supposed to be...  It's some sort of player stat, presumably (object 3) but that doesn't really explain why we're storing some constants to it.

Reply:
This was copied from the original game code, I didn't actually try to figure out what its doing. I'll leave a comment saying that at least.